pom.xml

```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.atlassian.jira.software.cloud.jenkins</groupId>
     <artifactId>atlassian-jira-software-cloud</artifactId>
-    <version>2.0.5-SNAPSHOT</version>
+    <version>2.0.6-SNAPSHOT</version>
```

Comment:
Should have been updated in the previous release ... the current live version is 2.0.5, so this is the snapshot version of 2.0.6

Reply:
Just curious, for prod release, will it be replaced to something non SNAPSHOT? 
Or it actually doesn't matter because it is all dockerised?

Reply:
@gxueatlassian yeah, for the release, the "-SNAPSHOT" prefix will be removed.